test/nodeshift-config-test.js

```diff
@@ -96,6 +96,51 @@ test('nodeshift-config no package.json', (t) => {
   });
 });
 
+test('nodeshift-config invalid "name" in package.json', (t) => {
+  const nodeshiftConfig = proxyquire('../lib/nodeshift-config', {
+    'openshift-config-loader': () => {
+      return Promise.resolve({
+        context: {
+          namespace: 'test-namespace'
+        },
+        cluster: 'http://mock-cluster'
+      });
+    },
+    'openshift-rest-client': () => { return Promise.resolve({}); }
+  });
+
+  const tmpDir = require('os').tmpdir()
```

Comment:
Wasn't certain about creating an item in `examples/` with an invalid name, so I took this approach instead. Happy to create a file in the repo if preferable.